core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -141,20 +142,22 @@
    private final ConfigurationManager configurationManager;
    private final String defaultCacheName;
 
-   @GuardedBy("this")
-   private boolean stopping;
+   private final Lock lifecycleLock = new ReentrantLock();
+   private volatile boolean stopping;
 
    /**
-    * Constructs and starts a default instance of the CacheManager, using configuration defaults.  See {@link org.infinispan.configuration.cache.Configuration Configuration}
-    * and {@link org.infinispan.configuration.global.GlobalConfiguration GlobalConfiguration} for details of these defaults.
+    * Constructs and starts a default instance of the CacheManager, using configuration defaults.  See {@link
```

Comment:
You should either revert these changes or change them so the link tags are on one line (and remove the labels, since they just duplicate the class name).